cardano-testnet/src/Testnet/Cardano.hs

```diff
@@ -207,64 +212,13 @@ cardanoTestnet testnetOptions H.Conf {..} = do
 
   let securityParam = 10
 
-  H.readFile configurationTemplate >>= H.writeFile configurationFile
-
-  forkOptions <- pure $ id
-    . HM.insert "EnableLogMetrics" (J.toJSON False)
-    . HM.insert "EnableLogging" (J.toJSON True)
-    . case cardanoEra testnetOptions of
-        Byron -> id
-          . HM.insert "LastKnownBlockVersion-Major" (J.toJSON @Int 1)
-
-        Shelley -> id
-          . HM.insert "TestShelleyHardForkAtEpoch" (J.toJSON @Int 0)
-          . HM.insert "LastKnownBlockVersion-Major" (J.toJSON @Int 2)
-
-        Allegra -> id
-          . HM.insert "TestShelleyHardForkAtEpoch" (J.toJSON @Int 0)
-          . HM.insert "TestAllegraHardForkAtEpoch" (J.toJSON @Int 0)
-          . HM.insert "LastKnownBlockVersion-Major" (J.toJSON @Int 3)
-
-        Mary -> id
-          . HM.insert "TestShelleyHardForkAtEpoch" (J.toJSON @Int 0)
-          . HM.insert "TestAllegraHardForkAtEpoch" (J.toJSON @Int 0)
-          . HM.insert "TestMaryHardForkAtEpoch" (J.toJSON @Int 0)
-          . HM.insert "LastKnownBlockVersion-Major" (J.toJSON @Int 4)
-
-        Alonzo -> id
-          . HM.insert "TestShelleyHardForkAtEpoch" (J.toJSON @Int 0)
-          . HM.insert "TestAllegraHardForkAtEpoch" (J.toJSON @Int 0)
-          . HM.insert "TestMaryHardForkAtEpoch" (J.toJSON @Int 0)
-          . HM.insert "TestAlonzoHardForkAtEpoch" (J.toJSON @Int 0)
-          . HM.insert "LastKnownBlockVersion-Major" (J.toJSON @Int 6)
```

Comment:
This code allowed us to choose which eras to hard fork to.  Are we dropping this support?

Reply:
We aren't. I was going to do a follow up PR implementing a function that will take an era and produce the "correct" yaml configuration value. I can do it here as well however.

Reply:
Either that or add a TODO so we don't forget.